docs/Community/SecurityAdvisoryHandling.md

```diff
@@ -0,0 +1,81 @@
+
+If OE SDK needs to issue a Security Advisory to address the reported concern, one of the security reviewers will:
+
+1. Create a new GitHub [Security Advisory (SA)](https://github.com/openenclave/openenclave/security/advisories) for OE SDK.
+   1. The description of the SA should accurately capture the nature, severity, and workarounds for the issue.
+      1. It does not need to capture the full technical details of the vulnerability or fix for brevity.
+2. Request a Common Vulnerability Enumeration (CVE) through GitHub.
+   1. This was previously done through MSRC as a CVE Numbering Authority (CNA), but OE has shifted to using GitHub as the CNA as a Linux Foundation project.
+3. Assign the appropriate collaborators to the SA, which includes:
+   1. The assignee(s) responsible for fixing the issue.
+   2. Any area owners or subject matter experts needed for review.
+   3. CI/CD coordinators for private testing and integration with the release. (e.g. @brmclaren).
+4. Create a temporary private fork to implement the code changes as part of the SA.
+   1. The assignee should follow the GitHub instructions and create an `advisory-fix` branch for their code changes.
+      1. The code changes should include an update to [CHANGELOG.md](/CHANGELOG.md) under the `### Security` subheader describing the issue addressed.
+   2. Once the changes are ready for review, they should be submitted as a PR to the master branch of the temporary fork.
+      1. GitHub does not currently appear to support protecting the master branch in temporary forks, so maintainers will need to enforce this process manually.
+5. Reach out to the CI/CD maintainers to establish a private CI/CD pipeline for testing code changes.
+
+By default, all SA are targeted for inclusion into the next release of the OE SDK, which usually happens quarterly.
+If the fix cannot be implemented or verified in time for an upcoming release, it can be included in a following patch version release.
+The security reviewers are responsible for deciding if a patch release is needed to address vulnerability before its scheduled public disclosure date.
+
+Once the security fix is reviewed, tested, and merged into the temporary fork master branch for release,
+the assignee and the release manager will coordinate integrating the fix into the release branch.
+This usually includes:
+
+1. Rebasing the temporary fork master branch to the project's master branch and testing the merged branch in private CI/CD.
+2. Publishing the SA and merging its master branch into the project master branch.
+   1. At this point, the vulnerability is considered publicly disclosed, and a new version of the OE SDK packages with the fix should be made available as quickly as possible.
+3. Cherry-picking the fix to the release branch.
+4. Building and releasing the OE SDK packages with the fix.
+5. Updating the published SA with a pointer to the released packages containing the fix.
```

Comment:
Please see early comments about avoiding delay between publishing the SA and providing released packages containing the fix.